modyn/selector/internal/selector_manager.py

```diff
@@ -135,6 +135,12 @@ def get_number_of_partitions(self, pipeline_id: int, trigger_id: int) -> int:
 
         return self._selectors[pipeline_id].get_number_of_partitions(trigger_id)
 
+    def get_available_labels(self, pipeline_id: int) -> list[int]:
```

Comment:
for this (and the other new functions), please add some unit tests as well. thank you!

Reply:
done